salt/addons/psp/manifests/10-podsecuritypolicy-privileged.yaml

```diff
@@ -38,7 +38,7 @@ spec:
     - azureDisk
     - azureFile
     - vsphereVolume
-  allowedFlexVolumes: []
+  #allowedFlexVolumes: []
```

Comment:
This is also set in 10-podsecuritypolicy-unprivileged.yaml - as far as I understand it, removing this has no effect on the policy (other than it being less explicit)

Reply:
Ok, I will just remove it in both files...

Reply:
Done.